src/components/common/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,39 @@
+import './navbar.css';
+import MainLogo from '../../../assets/icons/logo.svg';
+import Button from '../Button/Button';
+
+const Navbar = ({ user }) => {
+  const handleLoginButton = () => {
+    /** 후에 dom 연결 */
+  };
+
+  return (
+    <nav className="navbar">
+      <ul className="navbar-list">
+        <li className="navbar-item">
+          <a href="/" className="navbar-logo-link">
+            <img src={MainLogo} alt="Linkbrary Logo" />
+          </a>
+        </li>
+        <li className="navbar-item">
+          {!user ? (
+            <Button className="navbar-login-button" onClick={handleLoginButton}>
+              로그인
+            </Button>
+          ) : (
+            <div className="navbar-profile">
+              <img
+                className="navbar-userprofile-img"
+                src={user.profileImageSource}
+                alt="User Profile"
+              />
+              <span className="navbar-userprofile-email">{user.email}</span>
+            </div>
+          )}
```

Comment:
부정연산자보다는 긍정연산자로 처리하는게 보통 가독성에 더 좋은 것 같아요 :)
```suggestion
          {user ? (
            <div className="navbar-profile">
              <img
                className="navbar-userprofile-img"
                src={user.profileImageSource}
                alt="User Profile"
              />
              <span className="navbar-userprofile-email">{user.email}</span>
            </div>
          ) : (
            <Button className="navbar-login-button" onClick={handleLoginButton}>
              로그인
            </Button>
          )}
```

Reply:
저두 그렇게 생각합니다!
일반적으로도 코드의 가독성을 높이기 위해 긍정적인 조건을 사용하는 것이 좋다고 생각합니다. 
예전 자바스크립트 스타일 관련 설정을 했었을때 **Airbnb Javascript 스타일 가이드** 에서도 이와 관련된 권장 사항이 있더라구요! 
확실히 긍정적인 경로를 먼저 처리함으로서 코드의 의도를 더 분명하게 전달할 수 있는 것 같습니다!

해당 부분에 대해 더 주의를 기울이겠습니다!